### PR TITLE
Convert the last our of File.exists? to File.exist?

### PR DESCRIPTION
**Pull request by tas50**
Standardize on the new method

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Review by phiggins — APPROVED** [review]

**Comment by phiggins** [comment]
Looks like a bunch of unit tests are failing.

File: spec/support/shared/unit/provider/file.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def windows_path
 def setup_normal_file
   [ resource_path, normalized_path, windows_path].each do |path|
     allow(File).to receive(:file?).with(path).and_return(true)
-    allow(File).to receive(:exists?).with(path).and_return(true)
+    allow(File).to receive(:exist?).with(path).and_return(true)
     allow(File).to receive(:exist?).with(path).and_return(true)
```
Comment (lamont-granquist) [inline spec/support/shared/unit/provider/file.rb:47]: whole bunch of duplicated :exist? calls after this PR

File: spec/support/shared/unit/provider/file.rb [hunk 2]
```diff
@@ -533,7 +533,7 @@ def unlink
           provider.load_current_resource
           tempfile = double("Tempfile", path: "/tmp/foo-bar-baz")
           allow(content).to receive(:tempfile).and_return(tempfile)
-          expect(File).to receive(:exists?).with("/tmp/foo-bar-baz").and_return(true)
+          expect(File).to receive(:exist?).with("/tmp/foo-bar-baz").and_return(true)
```
Comment (lamont-granquist) [inline spec/support/shared/unit/provider/file.rb:536]: this one and the one below it smell like you may need to worry about ordering here

File: spec/support/shared/unit/provider/file.rb (hunk 1 above)
Comment (tas50) [reply spec/support/shared/unit/provider/file.rb]: Fixed

**Comment by tas50** [comment]
Closing this to let a new hire handle it instead

**Pull request closed without merging**